### PR TITLE
Add multiply function

### PR DESCRIPTION
**Pull request by xuan25**
resolves https://github.com/xuan25/python-calculator-group1/issues/4

### CODE REVIEW

**Review by YanyiPU — APPROVED** [review]
Marvellous!

**Comment by xuan25** [comment]
resolves #4

File: pythoncalculator/multiply.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```
Comment (YanyiPU) [inline pythoncalculator/multiply.py:2]: this should be 
```suggestion
    return x * y
```

**Review by YanyiPU — CHANGES_REQUESTED** [review]
这是一个伦理的问题

**Review by YanyiPU — APPROVED** [review]
这次就非常Marvellous

**Pull request merged**